CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.1.0]
+### Changed
+- fix: Add validation for types sign message primary type ([#350](https://github.com/MetaMask/eth-json-rpc-middleware/pull/350))
```

Comment:
This is a breaking change, right? We're throwing errors in this middleware in cases we weren't previously. There was no way the signature would _actually work_ in those cases though, so maybe this is fine to categorize as non-breaking 🤔

Reply:
Could you improve the change entry? The `fix:` prefix is confusing given that this is under the "Changed" category, and is a minor bump (which suggests that this is not just a bug fix).

Maybe we can describe this as "Improved validation of primary type for `signTypedDataV3 ` and `signTypedDataV34`"

Reply:
Sounds good, I updated the PR.